src/ledger/events.rs

```diff
@@ -0,0 +1,95 @@
+use chrono::DateTime;
+use chrono::Utc;
+use ethereum_types::U256;
+use revm::primitives::Address;
+use uuid::Uuid;
+
+use crate::eth::primitives::BlockNumber;
+use crate::eth::primitives::Hash;
+
+/// Represents token transfers (debits and credits) associated with a specific Ethereum account within a single transaction.
+///
+/// The `account_address` field identifies the main account involved in these transfers.
+///
+/// An event will be generated for each account involved in a transaction, meaning if a transaction is not the main account in this event,
+/// in another event it will be and credit and debit parties will be adjusted accordingly.
+pub struct AccountTransfersEvent {
```

Comment:
It always nice to have a schema_version field. Even if we don't need now and don't think will gonna need, better safe than sorry :) 

PS. maybe we can use kafka headers for this, don't know